frontend/frontend.php

```diff
@@ -88,6 +88,11 @@ public function __construct( &$links_model ) {
 		// Avoids the language being the queried object when querying multiple taxonomies
 		add_action( 'parse_tax_query', array( $this, 'parse_tax_query' ), 1 );
 
+		if ( $this->model->has_languages() ) {
```

Comment:
I believe this test is useless because we already did it before (we can't reach this point without languages): https://github.com/polylang/polylang/blob/3.7-beta3/include/class-polylang.php#L219
Also we could merge this action to the previous one.

Reply:
Indeed. Done in [eaf0254](https://github.com/polylang/polylang/pull/1650/commits/eaf02541d7cf7dcbc3c193a88e60b29eb095df7f).